src/main/java/com/example/postgresdemo/service/QuestionService.java

```diff
@@ -0,0 +1,63 @@
+package com.example.postgresdemo.service;
+
+import com.example.postgresdemo.exception.ResourceNotFoundException;
+import com.example.postgresdemo.model.Question;
+import com.example.postgresdemo.model.QuestionRequestDTO;
+import com.example.postgresdemo.model.QuestionResponseDTO;
+import com.example.postgresdemo.repository.QuestionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+@Service
+public class QuestionService {
+    @Autowired
+    private QuestionRepository questionRepository;
```

Comment:
Instead of field injection it's better to use constructor injection for dependencies. With constructor injection it will be easier to write unit tests for `QuestionService`

Reply:
Done